gcp/kubernetes/README.md

```diff
@@ -1,29 +1,43 @@
-# Kubernetes Configuration
+# jiant on GKE
 
 This directory contains configuration files and scripts for running a Kubernetes cluster on Google Kubernetes Engine (GKE) that can run our training and evaluation code. This is a convenient way to get an auto-scaling batch scheduler.
 
+You may be referring to this guide for several reasons:
+
+* You should probably check the prerequisites regardless: [Prerequisites](#Prerequisites)
+* If you are looking to start *from scratch*, including building a Docker image: [Creating a Docker Image](#creating-a-docker-image)
+* If you are looking to start a GKE cluster + NFS from an existing image: [Creating a GKE Cluster + NFS](#creating-a-gke-cluster--nfs)
+* If you are looking to access an existing GKE cluster for the first time: [Using an existing GKE cluster](#using-an-existing-gke-cluster)
+
 For more on Kubernetes, see [this comic](https://cloud.google.com/kubernetes-engine/kubernetes-comic/) and the [GKE quick-start guide](https://cloud.google.com/kubernetes-engine/docs/quickstart).
 
-All the instructions below assume Ubuntu Linux running on a Google Compute Engine (GCE) instance, but should also work on any Ubuntu machine with the Google Cloud SDK installed.
+======
+
+# Prerequisites
+
+### Workstation
 
-# Set-Up Instructions
+All setup instructions assume COS (Container OS) running on a Google Compute Engine (GCE) instance, but should also work on any Ubuntu machine with the Google Cloud SDK installed.
```

Comment:
Is this about the workstation or the container image? Clarify.

Reply:
Updated.